infra/token-transfer-server/src/lib/transfer.js

```diff
@@ -204,7 +202,6 @@ async function executeTransfer(transfer, opts) {
     case 'failed':
       transferStatus = enums.TransferStatuses.Failed
       eventAction = TRANSFER_FAILED
-      failureReason = 'Tx failed'
```

Comment:
why removing this? is it because it is redundant with the eventAction and does not add much more info?

Reply:
Yea